tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -461,6 +463,8 @@ IncludeByKey.SDW_AMP_FEEDBACK {
 									out_valid_bit_depth	32
 									out_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
 									out_ch_map	$CHANNEL_MAP_3_POINT_1
+									out_sample_type		$SAMPLE_TYPE_MSB_INTEGER
+									out_fmt_cfg		"$[($out_channels | ($out_valid_bit_depth * 256))]"
```

Comment:
This should have no practical impact given out_valid_bit_depth is 32. The bit-alignment only takes effect when valid-bits is smaller than the container size.

Reply:
> This should have no practical impact given out_valid_bit_depth is 32. The bit-alignment only takes effect when valid-bits is smaller than the container size.

Right, but it is better to be consistent. Other ALH dais set SAMPLE_TYPE_MSB_INTEGER no matter the valid_bit_depth is 32 bit or not. :)